.ci/acceptance-tests/inspec-integration.sh

```diff
@@ -60,7 +71,7 @@ bundle
 # Service accounts take several minutes to be authorized everywhere
 set +e
 
-for i in {1..50}
+for i in {1..30}
```

Comment:
What's the for loop for?

Reply:
Newly created service accounts are not reliably recognized by authentication servers immediately. So we keep trying until it works or has ran for enough time to say that something other than authentication is the issue. This is not ideal, but I don't see another option at this point.

Reply:
We've had some offline discussions about alternatives, and they're all terrible. This should be OK until we come up with something better.